src/main/scala/ch/openolitor/buchhaltung/BuchhaltungAktionenService.scala

```diff
@@ -207,14 +200,17 @@ class BuchhaltungAktionenService(override val sysConfig: SystemConfig) extends E
 
   def zahlungsEingangErledigen(meta: EventMetadata, entity: ZahlungsEingangModifyErledigt)(implicit personId: PersonId = meta.originator) = {
     DB localTxPostPublish { implicit session => implicit publisher =>
-      buchhaltungWriteRepository.getById(zahlungsEingangMapping, entity.id) map { eingang =>
+      buchhaltungWriteRepository.modifyEntity[ZahlungsEingang, ZahlungsEingangId](entity.id) { eingang =>
         if (eingang.status == Ok) {
           eingang.rechnungId map { rechnungId =>
             rechnungBezahlenUpdate(rechnungId, RechnungModifyBezahlt(eingang.betrag, eingang.gutschriftsDatum))
```

Comment:
Should we really execute external impacts within  modifyEntity operation or would it be better tomap it?
buchhaltungWriteRepository.updateEntity[ZahlungsEingang, ZahlungsEingangId](entity.id) {
 Map(
          zahlungsEingangMapping.column.erledigt -> true,
          zahlungsEingangMapping.column.bemerkung -> entity.bemerkung
       )
} map { eingang => 
 if (eingang.status == Ok) {
           eingang.rechnungId map { rechnungId =>
             rechnungBezahlenUpdate(rechnungId, RechnungModifyBezahlt(eingang.betrag, eingang.gutschriftsDatum))}
}